StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetDeferredValidator.swift

```diff
@@ -48,12 +49,41 @@ struct PaymentSheetDeferredValidator {
         }
         try validatePaymentMethodId(intentPaymentMethod: setupIntent.paymentMethod, paymentMethod: paymentMethod)
     }
-    
+
     static func validatePaymentMethodId(intentPaymentMethod: STPPaymentMethod?, paymentMethod: STPPaymentMethod) throws {
```

Comment:
Now that this no longer only validates the payment method id, we should rename it to match its new behavior.